morphic.js

```diff
@@ -4504,101 +4504,47 @@ CursorMorph.prototype.init = function (aStringOrTextMorph) {
 };
 
 CursorMorph.prototype.initializeClipboardHandler = function () {
-    // Add hidden text box for copying and pasting
+    // Add hidden text box for texting
     var myself = this;
 
     this.clipboardHandler = document.createElement('textarea');
     this.clipboardHandler.style.position = 'absolute';
     this.clipboardHandler.style.right = '101%'; // placed just out of view
```

Comment:
One suggestion: Instead of positioning the underlying `<textarea>` element outside the view port, you may want to put it at the same position of related `StringFieldMorph`, so that the IME's candidates panel can be shown near the input box. (Inspired by [CanvasInput](https://github.com/goldfire/CanvasInput/blob/master/CanvasInput.js#L127))

```javascript
var stringFieldBounds = this.target.parent.bounds;
this.clipboardHandler.style.left = stringFieldBounds.origin.x + 'px';
this.clipboardHandler.style.top  = stringFieldBounds.origin.y + 'px';
this.clipboardHandler.style.width = (stringFieldBounds.corner.x - stringFieldBounds.origin.x) + 'px';
this.clipboardHandler.style.height = (stringFieldBounds.corner.y - stringFieldBounds.origin.y) + 'px';
this.clipboardHandler.style.pointerEvents = 'none';
this.clipboardHandler.style.opacity = 0;
```